wazuh/wazuh-kibana-app
# Fix section deploy new agent

### Description

Fixed commands: Aix, OpenSUSE, Alpine, Suse11, Fedora, HP, Oracle Linux 5, Amazon Linux 2, Centos5
Changed the word 'or higher' in buttons to '+'.
Fixed validations for Hp, Solaris, and Alpine.
The enroll message for hp, solaris and Alpine os has been moved. The message now appears below the install and enroll section.
 
### Issues Resolved

#5042 

### Evidence

https://user-images.githubusercontent.com/99441266/209834280-cd31db31-602f-4061-99c5-d765ec6921aa.mov

### Steps to test

- Verify commands for Aix, OpenSUSE, Alpine, Suse11, Fedora, HP, Oracle Linux 5, Amazon Linux 2, Centos5. You can verify them [here](https://github.com/wazuh/wazuh-packages/issues/1625#issuecomment-1365838810)

- Verified that the buttons have changed the word 'or higher' to '+'.

- Alpine, Hp or Solaris do not allow configuring wazuh server, agentName and agentGroup.
When clicking on an operating system, set an incorrect agentName and then click on Alpine, Hp or Solaris verify that no validation error appears.

### Check List
